examples/containers/run.sh

```diff
@@ -22,8 +22,12 @@ DOCKER_USER=${DOCKER_USER:-couchbase}
 DOCKER_TAG=${DOCKER_TAG:-v1}
 CMOS_IMAGE=${CMOS_IMAGE:-$DOCKER_USER/observability-stack:$DOCKER_TAG}
 
-# Ensure we built the container locally first
-make -C "${SCRIPT_DIR}/../.." container
+# Ensure we build the container locally first otherwise make
+# sure one is tagged as above CMOS_IMAGE for use in the .env file.
+if [[ "${SKIP_CONTAINER_BUILD:-yes}" != "yes" ]]; then
+    echo "Building CMOS container"
+    make -C "${SCRIPT_DIR}/../.." container
```

Comment:
So the non-oss is effectively the default. What's a possible entrypoint where the container isn't locally built - if run.sh is run directly?